Source/Roads.py

```diff
@@ -78,6 +81,38 @@ def on_transfer(self, subject, parent, target):
                 return
         Core.Place.on_transfer(self, subject, parent, target)
 
+    def check_transfer_as_parent(self, subject, target):
```

Comment:
Du musst die Implementierung der Vater-Klasse nicht 1:1 kopieren, du kannst einfach schreiben:

    if not Core.Place.check_transfer_as_parent(self, subject, target):
        return False
    else:
        # deine Sachen

Du erweiterst die Funktionalitäten ja nur und ersetzt sie nicht.